bundles/org.openhab.binding.ipcamera/src/main/java/org/openhab/binding/ipcamera/internal/handler/IpCameraHandler.java

```diff
@@ -525,6 +525,18 @@ private void checkCameraConnection() {
             }
             return;// ffmpeg snapshot stream is still alive
         }
+
+        // if ONVIF cam also use connection state which is updated by regular messages to camera
+        if (thing.getThingTypeUID().getId().equals(ONVIF_THING) && snapshotUri.isEmpty()) {
+            logger.trace("is ONVIF with empty snapshot URI");
```

Comment:
This will create log entries way too often, if you want to log this is occuring, then consider placing a log comment on the first time the camera comes ONLINE and not every time the binding checks that the camera is still connected.